.teamcity/src/main/kotlin/promotion/BasePromotionBuildType.kt

```diff
@@ -43,6 +43,9 @@ abstract class BasePromotionBuildType(vcsRootId: String, cleanCheckout: Boolean
             param("env.GRADLE_CACHE_REMOTE_PASSWORD", "%gradle.cache.remote.password%")
             param("env.GRADLE_CACHE_REMOTE_URL", "%gradle.cache.remote.url%")
             param("env.GRADLE_CACHE_REMOTE_USERNAME", "%gradle.cache.remote.username%")
+            // https://www.jetbrains.com/help/teamcity/shared-resources.html#Viewing+Shared+Resources+Usage
+            // we only allow 1 promotion job running at the same time to avoid website xml conflicts
+            param("teamcity.locks.writeLock.promotionJobLock", "PromotionJobLock")
```

Comment:
I think I would name the lock differently. Since the resource we want to protect is the website xml, I think that is how the lock/resource should be called. How about `WebsiteReleasesXml`?